manifests/charts/ai-gateway-helm/templates/admission_webhook.yaml

```diff
@@ -0,0 +1,90 @@
+# Copyright Envoy AI Gateway Authors
+# SPDX-License-Identifier: Apache-2.0
+# The full text of the Apache license is available in the LICENSE file at
+# the root of the repo.
+
+apiVersion: admissionregistration.k8s.io/v1
+kind: MutatingWebhookConfiguration
+metadata:
+  name: envoy-ai-gateway-gateway-deployment-mutator
+webhooks:
+  - name:  {{ include "ai-gateway-helm.controller.fullname" . }}.{{ .Release.Namespace }}.svc.cluster.local
+    clientConfig:
+      service:
+        name:  {{ include "ai-gateway-helm.controller.fullname" . }}
+        namespace: {{ .Release.Namespace }}
+        path: /mutate
+      {{- if .Values.controller.mutatingWebhook.caBundle }}
+      caBundle: {{ .Values.controller.mutatingWebhook.caBundle }}
```

Comment:
i am no expert in this so i am not sure what's the standard way to expose `MutatingWebhookConfiguration` configuration in helm. i need your inputs @yuzisun @arkodg

Reply:
We use cert manager to automatically inject the ca bundle